e2etest/l2tests/interface_selector.go

```diff
@@ -83,6 +85,40 @@ var _ = ginkgo.Describe("L2-interface selector", func() {
 			framework.ExpectNoError(err)
 		})
 
+		ginkgo.It("Validate ServiceL2Status interface", func() {
+			ginkgo.By("generate a random int to choose some interface for announcing")
+			i := rand.Intn(len(NodeNics))
+			resources := config.Resources{
+				L2Advs: []metallbv1beta1.L2Advertisement{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "with-interfaces",
+						},
+						Spec: metallbv1beta1.L2AdvertisementSpec{
+							Interfaces: []string{NodeNics[i]},
+						},
+					},
+				},
+			}
+			err := ConfigUpdater.Update(resources)
+			framework.ExpectNoError(err)
+
+			svc, _ := service.CreateWithBackend(cs, f.Namespace.Name, "lb-service")
+			defer func() {
+				err := cs.CoreV1().Services(svc.Namespace).Delete(context.TODO(), svc.Name, metav1.DeleteOptions{})
+				framework.ExpectNoError(err)
+			}()
+
+			var l2Status *metallbv1beta1.ServiceL2Status
+			gomega.Eventually(func() error {
+				var err error
+				l2Status, err = status.GetL2Status(ConfigUpdater.Client(), svc)
+				return err
+			}, 2*time.Minute, 1*time.Second).ShouldNot(gomega.HaveOccurred())
```

Comment:
can you add a 5s `gomega.Consistently` after this to make sure the speakers do not fight?